LoggingKata.Test/TacoParserTests.cs

```diff
@@ -24,6 +37,14 @@ public void ShouldParse(string str)
         public void ShouldFailParse(string str)

```

Comment:
There are more examples of bad input data that could be used as InlineData for this method. For example, "10, ten, Taco Bell", "10, Taco Bell", etc.